local_repo/roles/pulp_validation/tasks/check_pulp_status.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Gather facts on Pulp container
+  containers.podman.podman_container_info:
+    name: "{{ pulp_container_name }}"
+  register: container_info
+
+- name: Print Container Status
+  debug:
+    msg: "{{ container_info.containers[0].State.Status }}"
+
+- name: Pulp container status
+  ansible.builtin.debug:
+    msg: "Pulp container Status: {{ container_info.containers[0].State.Status }}"
```

Comment:
all comments should be in variable and unwanted debug statement can be kept with verbose level option 2